packages/dds/tree/src/test/domains/json/jsonCursor.spec.ts

```diff
@@ -75,6 +77,12 @@ describe("JsonCursor", () => {
             assert.deepEqual([...new JsonCursor(false).keys], []);
             assert.deepEqual([...new JsonCursor(true).keys], []);
         });
+
+        it("null", () => {
+            // Rationale: While Object.keys(..) does not accept 'null', the SharedTree data model must
+            //            as any value may be augmented with by the view schema.
```

Comment:
I don't like this indent style (leads to more line wrapping, and doesn't have great tooling support ad doesn't seem to add value in my opinion, which I'll admit is subjective: keep it if you want)

Also I think this reasoning is wrong. This has nothing to do with shared tree and its data model, and is entirely a property of JsonCursors allowed set of inputs and the JsonDomain it works with. Nothing in shared tree would be broken by excluding null from the json domain, or requiring it to have some child (those would just be silly choices for this specific domain)
```suggestion
            // JsonCursor accepts all valid json, which includes null. Ensure null correctly lists no keys:
```